app/src/main/java/wishcantw/vocabulazy/audio/VLTextToSpeech.java

```diff
@@ -128,6 +135,14 @@ boolean isLanguageAvailable() {
         return (mTextToSpeech.isLanguageAvailable(Locale.ENGLISH) >= 0 && mTextToSpeech.isLanguageAvailable(Locale.TAIWAN) >= 0);
     }
 
+    public void mute() {
+        AppPreference.getInstance().setPlayerVolume(0.0f);
+    }
+
+    public void unMute() {
+        AppPreference.getInstance().setPlayerVolume(1.0f);
+    }
+
```

Comment:
Sorry I didn't see the additional question about play time, and yes, currently users are able to listen to our app continuously up to 40 minutes